specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/connection.management/reset.http2.stream/client.rpt

```diff
@@ -48,14 +48,16 @@ read [0x00 0x00 0x00]                   # length = 0
      [0x01]                             # ACK
      [0x00 0x00 0x00 0x00]              # stream_id = 0
 
-write [0x00 0x00 0x13]                  # length = 19
-      [0x01]                            # HEADERS frame
-      [0x04]                            # END_HEADERS
-      [0x00 0x00 0x00 0x01]             # stream_id = 1
-      [0x83]                            # :method: POST
-      [0x86]                            # :scheme: http
-      [0x84]                            # :path: /
-      [0x01] [0x0e] "localhost:8080"    # :authority: localhost:8080
+write [0x00 0x00 0x33]                                      # length = 51
+      [0x01]                                                # HEADERS frame
+      [0x04]                                                # END_HEADERS
+      [0x00 0x00 0x00 0x01]                                 # stream_id = 1
+      [0x83]                                                # :method: POST
+      [0x86]                                                # :scheme: http
+      [0x84]                                                # :path: /
+      [0x01] [0x0e] "localhost:8080"                        # :authority: localhost:8080
+      [0x0f 0x10] [0x18] "text/plain;charset=UTF-8"         # content-type
+      [0x0f 0x0d] [0x02] "13"                               # content-length
```

Comment:
Is this scenario now a duplicate of one of the various `client.sent.rst.stream.*` scenarios?
If so, let's remove this scenario and use the more specifically named equivalent scripts.

Reply:
Yes